cmd/gctsCloneRepository.go

```diff
@@ -0,0 +1,101 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/cookiejar"
+
+	gabs "github.com/Jeffail/gabs/v2"
```

Comment:
not sure if we should add this dependency.
Why is this needed?

Reply:
Gabs is needed for JSON dynamic parsing. It makes your life much easier and the code more readable.